src/olympia/devhub/templates/devhub/addons/submit/wizard.html

```diff
@@ -0,0 +1,86 @@
+{% extends "devhub/addons/submit/base.html" %}
+
+{% block title %}
+{{ dev_page_title(_('Theme generate'), addon) }}
+{% endblock %}
+
+{% block primary %}
+  <h3>{{ _('Theme generator') }}</h3>
+  <div id="theme-wizard">
+    <div>
+        <h3>Theme name</h3>
+        <input type="text" id="theme-name"/>
+    </div>
+    <div id="theme-header" class="row">
+      <label class="row" for="header-img">
+        {{ _('Select a header image for your Theme.') }}
+        <span class="req" title="{{ _('required') }}">*</span>
+      </label>
+      <input data-allowed-types="image/jpeg|image/png|image/apng|image/svg|image/gif"
+             id="header-img" name="header" type="file">
+      <ul class="note">
+        <li>{{ _('Image height should be 200 pixels') }}</li>
+        <li>{{ _('PNG, JPG, APNG, SVG or GIF (not animated)') }}</li>
+        <li>{{ _('Aligned to top-right') }}</li>
+      </ul>
+      <ul class="errorlist"></ul>
+      <img class="preview" alt="">
+      <a href="#" class="reset">
+        {{ _('Select a different header image') }}</a>
+    </div>
+    <div class="colors">
+      <h3>{{ _('Select colors for your Theme.') }}</h4>
+      {% set colors = [
```

Comment:
Kinda sad it's in the template and not the view, but since it's the regular upload view, ok.